parser/bench_test.go

```diff
@@ -17,34 +17,103 @@ import (
 	"testing"
 )
 
-func BenchmarkSysbenchSelect(b *testing.B) {
+func sysebchSelect(parser *Parser, sql string, num int, b *testing.B) {
+	for i := 0; i < num; i++ {
+		_, err := parser.Parse(sql, "", "")
+		if err != nil {
+			b.Fatal(err)
+		}
+	}
+}
+
+func BenchmarkSysbenchSelect1024(b *testing.B) {
```

Comment:
We don't need multiple benchmarks to run a different number of loops.

Reply:
The purpose is to simulate different workloads. My way to do this is wrong?

Reply:
The workloads are not different.

Reply:
but `testing.B` thinks it is different workload. `testing.B` will run enough time to make sure the accurateness. If we us this, `testing.B` thinks its a operation. In my understanding, it is different workload. If my understanding is wrong, then what is correct way to do it?

Reply:
If you want the benchmark run longer to get a more accurate result, you can specify the time to run. like `-benchtime=10s`.